djangocms_admin_style/sass/mixins/_custom.scss

```diff
@@ -19,66 +19,70 @@
     background-size: 100% 400%;
 }
 
-@mixin button-common-init {
-    border: 1px solid $gray-lighter !important;
-    background-color: $white !important;
-    background-image: none !important;
-    box-shadow: none !important;
-    padding: 10px 15px !important;
-    &:hover {
-        background: $gray-lighter !important;
+// taken from bootstrap with adaptations
+@function important($important) {
+    @if($important == true) {
+        @return !important;
+    } @else {
+        @return true;
     }
 }
+@mixin button-variant($color, $background, $border, $important: false) {
+    background-image: none important($important);
+    margin-bottom: 0; // For input.btn
+    padding: 10px 30px important($important);
+    border-radius: $btn-border-radius-base important($important);
+    color: $color important($important);
+    font-weight: normal important($important);
+    font-size: $font-size-normal important($important);
+    background-color: $background important($important);
+    border: 1px solid $border important($important);
 
-@mixin button-common() {
-    @include button-common-init();
-    overflow: visible;
-    color: $gray !important;
-    font-size: 18px !important;
-    font-weight: normal !important;
-    line-height: 21px !important;
-    text-decoration: none !important;
-    width: auto;
-    border-radius: $border-radius-base;
-    &.small {
-        padding: 3px 8px 4px !important;
-    }
-    &:hover,
-    &:active,
-    &.select,
-    &:visited {
-        color: $gray;
-        text-decoration: none;
-        background: $gray-lighter;
+    &:focus,
+    &.focus {
+        color: $color important($important);
+        background-color: darken($background, 10%) important($important);
+        border-color: darken($border, 25%) important($important);
+        text-decoration: none important($important);
     }
     &:hover {
-        border: solid 1px darken($gray-lighter, 12%) !important;
-    }
-    &.animate {
-        @include stripes(0.2);
+        color: $color important($important);
+        background-color: darken($background, 10%) important($important);
+        border-color: darken($border, 12%) important($important);
+        text-decoration: none important($important);
     }
-}
+    &:active,
+    &.cms-btn-active {
```

Comment:
it might make sense to slightly modify this, i'm pretty sure we can live without `cms-` prefixed classes in the admin